Makefile

```
@@ -41,6 +42,30 @@ cpm: ## Tier 2: lint + complexity + tests (<60s, pre-push)
 cpm-full: ## Tier 3: everything — dead-code, xref, e2e (CI)
 	@bash lib/cpm/shell/cpm-check.sh full
 
+##@ Workflow (daily flow: code → check → push → PR → merge)
+
+workflow: ## Show the recommended workflow steps
```

checkmake (0.3.2)

[warning] Target body for "workflow" exceeds allowed length of 5 lines (12).

(maxbodylength)